backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/NamespaceImpl.java

```diff
@@ -274,21 +278,25 @@ public Membership resolve(String qualifiedName) {
      */
     @Override
     public Membership resolveGlobal(String qualifiedName) {
+        Membership founded = null;
+
+        Resource owningEcoreResource = this.eResource();
+        if(owningEcoreResource != null){
+            ResourceSet owningResourceSet = owningEcoreResource.getResourceSet();
+            if(owningResourceSet != null){
+                for(var resource : owningResourceSet.getResources()){
+                    EObject root = resource.getContents().get(0);
+                    if(root instanceof Namespace rootNamespace){
+                        founded = rootNamespace.resolveVisible(qualifiedName);
+                    }
+                    if(founded != null){
+                        break;
```

Comment:
I think the use of break statement is prohibited

Reply:
Checkstyle is not called in the generated code, so It is ok.
You can still try to remove it if you want.

Reply:
I was not sure :)

Reply:
Ok, nothing to do so